servers/text/text_server.h

```diff
@@ -458,6 +474,20 @@ class TextServer : public RefCounted {
 	virtual Vector2 get_hex_code_box_size(int64_t p_size, int64_t p_index) const;
 	virtual void draw_hex_code_box(const RID &p_canvas, int64_t p_size, const Vector2 &p_pos, int64_t p_index, const Color &p_color) const;
 
+	/* Draw list interface */
+
+	virtual RID create_draw_list() = 0;
+	virtual void draw_list_sort(const RID &p_dc) = 0;
+	virtual void draw_list_reserve(const RID &p_dc, int64_t p_new_items) = 0;
+
+	virtual void draw_list_add_hexbox(const RID &p_dc, int64_t p_layer, const Transform2D &p_transform, const Vector2 &p_pos, int64_t p_index, int64_t p_size, const Color &p_modulate) const = 0;
+	virtual void draw_list_add_rect(const RID &p_dc, int64_t p_layer, const Transform2D &p_transform, const Rect2i &p_rect, bool p_filled, const Color &p_modulate) const = 0;
```

Comment:
The `p_rect` needs to be Rect not Recti, or it won't be correct. See the search highlight in the script editor:

<img width="78" height="57" alt="image" src="https://github.com/user-attachments/assets/1a088c7d-5edf-4217-9b06-2ba50948bddc" />

This is probably also true for `add_line`'s points and `add_texture`'s `p_dst_rect`.